rpc/client/client.go

```diff
@@ -433,8 +433,28 @@ func (c *Client) Commit(ctx context.Context, height *int64) (*ctypes.ResultCommi
 	return ctypes.NewResultCommit(&block.Header, commit, true), nil
 }
 
-func (c *Client) Validators(ctx context.Context, height *int64, page, perPage *int) (*ctypes.ResultValidators, error) {
-	panic("Validators - not implemented!")
+func (c *Client) Validators(ctx context.Context, heightPtr *int64, pagePtr, perPagePtr *int) (*ctypes.ResultValidators, error) {
```

Comment:
Do we want to use the term Aggregator, or are we wanting to maintain compatability with tendermint rpc and keep using Validator?

Reply:
Created follow-up issue: https://github.com/celestiaorg/optimint/issues/289